reference/OpenAPI.json

```diff
@@ -24971,6 +24970,7 @@
             "description": "Flag indicating whether this voucher is a referral code; `true` for campaign type `REFERRAL_PROGRAM`."
           },
           "voucher": {
+            "type": "object",
```

Comment:
why?

Reply:
Błąd kopiowania z OpenAPIwebhooks.json

Tyle że ja tego jeszcze nie dałem do review i bym to poprawił jutro :)